jest.config.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
   setupFiles: ['<rootDir>/.jest/register-context.js', 'jest-canvas-mock'],
   setupFilesAfterEnv: ['<rootDir>/.jest/moment.js'],
   transform: {
-    '^.+\\.stories\\.js$': '@storybook/addon-storyshots/injectFileName',
+    '^.+\\.mdx?$': '@storybook/addon-docs/jest-transform-mdx',
```

Comment:
Storybook storyshot is not able to pick mdx stories at the moment. This is a known issue in Storybook Vue, we have to waiting for official solution.